src/app/pages/ArticlePage/index.test.tsx

```diff
@@ -141,6 +141,44 @@ afterAll(() => {
 });
 
 describe('Article Page', () => {
+  it.each([
+    {
+      title:
+        'should show the CTA on non Lite Site pages, when the toggle is enabled',
+      isLite: false,
+      liteSiteCTA: true,
+      expected: true,
```

Comment:
Can we rename these variables to be more descriptive of what they would do?

```suggestion
      toggleEnabled: true,
      shouldBeDisplayed: true,
```